app/components/user-avatar.hbs

```diff
@@ -6,4 +6,7 @@
   title={{this.title}}
   decoding="async"
   ...attributes
-/>
\ No newline at end of file
+/>
+{{#if this.sudo}}
+  🧙
+{{/if}}
```

Comment:
... but I'm actually not convinced that the `UserAvatar` component is the right place for this. I think I'd rather prefer this in the `Header` component since the `UserAvatar` component is used in a couple of other places where the sudo icon does not make much sense.

Reply:
Fair enough. Moved.